### DISCUSSION TITLE
Extra line from narrowers API response causes N3 parse error

### DISCUSSION THREAD

**Discussion by luoenzhen** [RDFLib/prez#309] [Q&A]
On prez v3, when I call `/v/vocab/{concept_scheme_curie}/{concept_curie}/narrowers` and get response with extra line below the prefix section as:
```
@prefix dcterms: <http://purl.org/dc/terms/> .
@prefix prez: <https://prez.dev/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .

<:5699eca7-9ef0-47a6-bcfb-9306e0e2b85e> dcterms:identifier "ns15:5699eca7-9ef0-47a6-bcfb-9306e0e2b85e"^^prez:identifier .

<http://linked.data.gov.au/def/tern-cv/2ffe4363-dea1-4188-9f8f-835983d47fe2> a skos:Concept ;
    rdfs:label "biological parameter"@en ;
    dcterms:identifier "trn-cv:2ffe4363-dea1-4188-9f8f-835983d47fe2"^^prez:identifier ;
    skos:definition "Living organisms and life." ;
...
``` 

That line causes n3 parses error with `parsing N3 data: Error: Invalid IRI on line 8`.
I've compared that vocab with others seems no difference on the data structure and responses from other vocabs don't have that line.

Can someone give me some insight on how to resolve this issue please? Thanks.

**Comment by recalcitrantsupplant** [comment]
The iri <:5699eca7-9ef0-47a6-bcfb-9306e0e2b85e> is probably invalid (missing a schema), it's at least not a "good" iri; this is not a prez specific issue, check elsewhere for information on creating valid iris.
